guides/common/modules/proc_adding-a-vnc-connection-for-a-managed-host-on-proxmox.adoc

```diff
@@ -0,0 +1,43 @@
+[id="Adding_a_VNC_Connection_for_a_Managed_Host_on_Proxmox_{context}"]
+= Adding a VNC Connection for a Managed Host on Proxmox
+
+Use this procedure to add a VNC connection for a managed host on Proxmox.
+
+.Limitation
+* This procedure only works for hosts based on `qemu`.
+It does not work for LXC container on Proxmox.
+
+.Procedure
+. On your Proxmox, open the required port for VNC:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# firewall-cmd --add-port=_My_Port_/tcp --permanent
```

Comment:
Can't firewall-cmd add a port range?

Reply:
yes, than its `--add-port=X-Y`

Reply:
This procedure is aimed at individual managed hosts. Else, you'd tell users to open ports that are not required.

I would not change this.